README.md

```diff
@@ -45,6 +45,16 @@ Make sure Node.js is set up on your machine. If you are on the CS-RDSH-02 virtua
 - Then, tell nvm that you want to use that verion: `nvm use node`.
   To use a specific Node version - usually the latest Long Term Support (LTS) version - replace `node` in the above commands with the desired version or alias, e.g. `nvm install 14.15.4` for version 14.15.4 or `nvm install --lts` for the latest LTS.
 
+#### MacOS:
+
+- Follow the linux installations. 
+- After installing nvm make sure you do quit terminal and then open it back. 
+- To check if the `nvm` is install in your macOS type this command `nvm -v`.
+- Next you can install the latest version of the nvm you can do it by typing this command nvm `nvm install --lts` for the most recent stable version which means you will not worry to update it for a while.
```

Comment:
I think this addition is redundant after #1153. Since the Linux and Mac instructions are the same (and have been updated to state as much), this isn't needed.